deploy/eck-operator/templates/configmap.yaml

```diff
@@ -79,6 +79,6 @@ data:
     {{- if not .Values.config.containerSuffix }}
     ubi-only: {{ .Values.config.ubiOnly }}
     {{- end }}
-    {{- with .Values.webhook.secret }}
+    {{- with .Values.webhook.certsSecret }}
```

Comment:
Even if we fix this I don't understand how this can be used today since the `WEBHOOK_SECRET` env. variable seems to take precedence over `webhook-secret` in the configmap : https://github.com/elastic/cloud-on-k8s/blob/233cbd07a6b2a2e5dd70bba27e3ff6efaf3cc964/deploy/eck-operator/templates/statefulset.yaml#L73-L76